src/App.tsx

```diff
@@ -8,6 +8,7 @@ import themeFile from "./util/theme";
 import home from "./pages/home";
 import login from "./pages/login";
 import signup from "./pages/signup";
+import chain from "./pages/chains/chain";
```

Comment:
Gotta wonder why these components are not capitalized. This is against naming conventions.